### PR TITLE
Improve p2p

### PR DESCRIPTION
**Pull request by polarker**

### CODE REVIEW

File: flow/src/main/scala/org/alephium/flow/network/DiscoveryServerState.scala [hunk 1]
```diff
@@ -144,6 +145,15 @@ trait DiscoveryServerState {
     pendings --= deadPendings
   }
 
+  def setUnreachable(remote: InetSocketAddress): Unit = {
+    unreachables += remote -> TimeStamp.now().plusMinutesUnsafe(10)
+    remove(remote)
+  }
+
+  def mightReachable(remote: InetSocketAddress): Boolean = {
```
Comment (tdroxler) [inline flow/src/main/scala/org/alephium/flow/network/DiscoveryServerState.scala:154]: naming sounds weird, simply `isReachable` or `mightReach` no?

File: flow/src/main/scala/org/alephium/flow/core/BlockFlow.scala [hunk 2]
```diff
@@ -292,7 +292,7 @@ object BlockFlow extends StrictLogging {
     }
 
     def updateBestDepsUnsafe(): AVector[TransactionTemplate] =
-      (brokerConfig.groupFrom until brokerConfig.groupUntil).foldLeft(
+      brokerConfig.groupRange.foldLeft(
```
Comment (tdroxler) [inline flow/src/main/scala/org/alephium/flow/core/BlockFlow.scala:295]: sweet

**Review by tdroxler — APPROVED** [review]
LGTM nice job

File: flow/src/main/scala/org/alephium/flow/network/DiscoveryServerState.scala (hunk 1 above)
Comment (polarker) [reply flow/src/main/scala/org/alephium/flow/network/DiscoveryServerState.scala:154]: it is might be reachable, but we are not sure in this case. If it's false, then we are sure that it's not reachable. I don't know a better name for this

**Pull request merged**